CMakeLists.txt

```diff
@@ -479,6 +479,25 @@ ELSE()
 ENDIF()
 
 
+# Find the libdap package so that Aspect can connect to the OPeNDAP servers
+# Author: Kodi Neumiller
+# Added: 10/16/18
+IF(LIBDAP)
+  LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}/cmake/modules")
+  MESSAGE(STATUS "cmake module path = ${CMAKE_MODULE_PATH}")
+  FIND_PACKAGE(LIBDAP)
+  IF(${LIBDAP_FOUND})
+    FIND_PACKAGE(LibXml2)
+    FIND_PACKAGE(CURL)
+    INCLUDE_DIRECTORIES(${LIBDAP_INCLUDE_DIR} ${LIBXML2_INCLUDE_DIR} ${LIBXML2_INCLUDE_DIR}/libxml2 ${CURL_INCLUDE_DIR})
+    TARGET_LINK_LIBRARIES(${TARGET} ${LIBDAP_LIBRARIES} ${LIBXML2_LIBRARIES} ${CURL_LIBRARIES})
+    ADD_DEFINITIONS(-DHAVE_LIBDAP=TRUE)
```

Comment:
can you use something like https://github.com/geodynamics/aspect/blob/master/CMakeLists.txt#L441 instead please?